Source/Fuse.GeoLocation/iOS/IOSLocationProvider.uno

```diff
@@ -109,6 +109,33 @@ namespace Fuse.GeoLocation
 			StartUpdatingLocation(_lm);
 		}
 
+
+		[Foreign(Language.ObjC)]
+		public bool IsLocationEnabled() 
+		@{
+			//check is hardware enabled
+			if ([CLLocationManager locationServicesEnabled]) {
+
+				//check user authorization
+				switch ([CLLocationManager authorizationStatus]) {
+					case kCLAuthorizationStatusDenied: return false;
+						break;
+					case kCLAuthorizationStatusRestricted: return false;
+						break;
+					case kCLAuthorizationStatusAuthorizedAlways: return true;
+						break;
+					case kCLAuthorizationStatusAuthorizedWhenInUse: return true;
+						break;
+					case kCLAuthorizationStatusNotDetermined: return false;
+						break;
+					default: return false;
+						break;
+				}
```

Comment:
You don't need breaks after the return-statements.
But, I think this whole check would be cleaner if written like this:
```java
switch ([CLLocationManager authorizationStatus]) {
case kCLAuthorizationStatusAuthorizedAlways:
case kCLAuthorizationStatusAuthorizedWhenInUse:
        return true;

default:
        return false;
}
```

Point is, you only need to distinguish between the true and false case, and since the default-case already covers false, we can get rid of all other false-cases. Then finally we can merge the two true-cases.

Reply:
I guess I was being a little verbose, just thought the next guy to read the cases could see the reasons but I guess its not important